docker-images/java-base/scripts/launch_java.sh

```diff
@@ -16,6 +16,6 @@ export MALLOC_ARENA_MAX=2
 JAVA_OPTS="${JAVA_OPTS} -Dvertx.cacheDirBase=/tmp -Djava.security.egd=file:/dev/./urandom"
 
 # Enable GC logging for memory tracking
-JAVA_OPTS="${JAVA_OPTS} -XX:NativeMemoryTracking=summary -verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps"
+JAVA_OPTS="${JAVA_OPTS} ${EO_GC_LOG_OPTS}"
```

Comment:
`EO_GC_LOG_OPTS` in the common `java-base` module? Surely better to name it something like `STRMZI_GC_LOG_OPTS` if you're putting it in `java-base`?